langchain-core/src/runnables/tests/runnable_remote.test.ts

```diff
@@ -215,4 +215,20 @@ describe("RemoteRunnable", () => {
       AIMessage
     );
   });
+
+  test("StreamIterator yield stream model output", async () => {
+    const remote = new RemoteRunnable({ url: `${BASE_URL}/b` });
+    const streamIterator = await remote._streamIterator({ text: "What are the 5 best apples?" });
```

Comment:
This should test streaming usage in a chain though?

`const chain = prompt.pipe(remote).stream();`

Reply:
I thought I would just test `_streamIterator` function which I added. But, I think what you said make sense. Let me try with chain.

Reply:
Updated!!